### PR TITLE
Add form validation to the certification page.

### PR DESCRIPTION
**Pull request by idealisms**
Makes most inputs required and add some patterns
for things that should be numbers.

The styling needs some work, but I'll wait until we
get the designs for it.

We're still submitting the fake data though
(can be fixed in a separate change).

===

- [X] Snapshots updated, if output HTML/JS has changed (`npm run test:update-snapshots`)
- [ ] Changes reviewed on mobile using devtools, if output HTML/CSS has changed
- [ ] Spanish checked by adding `?lng=es` to URL e.g. `/guide/benefits?lng=es`, if Spanish updated
- [ ] Automated tests added, if new functionality added
- [ ] Documentation (e.g. READMEs) updated, if relevant
- [ ] IE11 and Edge compatibility confirmed via manual testing in Browserstack, if new libraries added or newish JS/HTML/CSS features used for the first time in this codebase
- [ ] Accessibility & performance audit run using Google Lighthouse, if major changes made

screencapture-localhost-3000-retroactive-certification-certify-2020-02-08-2020-06-19-13_58_02

### CODE REVIEW

File: src/client/App.scss [hunk 1]
```diff
@@ -99,6 +99,7 @@ footer {
 }
 
 // This removes the green border around validated fields.
-.form-control.is-valid, .was-validated .form-control:valid {
+.form-control.is-valid,
+.was-validated .form-control:valid {
```
Comment (idealisms) [inline src/client/App.scss:103]: This was reformatted by prettier.

File: src/client/components/EmployersQuestions/index.js [hunk 2]
```diff
@@ -121,15 +140,23 @@ function EmployerQuestions(props) {
             value={lastDateWorkedDay}
             maxLength={2}
             onChange={(e) => handleDateChange("day", e.target.value)}
+            required
+            pattern="0?[1-9]|1\d|2\d|3[01]"
```
Comment (idealisms) [inline src/client/components/EmployersQuestions/index.js:144]: This isn't that smart. You could do Feb 30 or Apr 31.

File: src/client/components/EmployersQuestions/index.js [hunk 3]
```diff
@@ -47,15 +47,28 @@ function EmployerQuestions(props) {
     dateTokens.length === 3 ? dateTokens[1] : ""
   );
 
-  function renderTextInput(name, className) {
+  /**
+   * Create a text input element.
+   * @param {string} name The name for the employer data field.
+   * @param {Object?} options Optional controls:
+   *     className: The classname for applying styles.
```
Comment (sharonwarner) [inline src/client/components/EmployersQuestions/index.js:54]: Ugh, these alerts are so annoying! Do you mind adding that option to the eslint file?

File: src/client/components/EmployersQuestions/index.js (hunk 2 above)
Comment (sharonwarner) [reply src/client/components/EmployersQuestions/index.js:144]: I think that's fine!

File: src/client/components/EmployersQuestions/index.js (hunk 3 above)
Comment (idealisms) [reply src/client/components/EmployersQuestions/index.js:54]: Oops, fixed.

**Pull request merged**